modules/ROOT/pages/getting-started-proxy.adoc

```diff
@@ -48,42 +52,49 @@ then API Manager displays APIs to choose from. Select the API one that was recen
 +
 - *Managing Type*: You may choose between `Basic Endpoint` or `Endpoint with Proxy`. Here we are using the endpoint with proxy.
 +
```

Comment:
Tip or something explaining what a Basic endpoint is?

Reply:
Again, what is "to configure your API with proxy" vs. the earlier wording of "deploy your proxy API"?
Clarify the difference between wordings or standardize to one most precise wording throughout.